scripts/install.sh

```diff
@@ -378,7 +378,7 @@ wait_for_pods_ready "-l control-plane=modelmesh-controller"
 
 # Older versions of kustomize have different load restrictor flag formats.
 # Can be removed once Kubeflow installation stops requiring v3.2.
-kustomize_version=$(kustomize version --short | grep -o -E "[0-9]\.[0-9]\.[0-9]")
+kustomize_version=$(kustomize version --short | grep -o -E "[0-9]\.[0-9]\.[0-9]|\(devel\)")
```

Comment:
Can you confirm that the logic below for the `kustomize_load_restrictor_arg` continues to work with the change? Seems to me like with kustomize version `(devel)`, I hit `Error: unknown flag: --load_restrictor none`.

Reply:
@rafvasq The output of `kustomize version --short` on my machine is:
```
Flag --short has been deprecated, and will be removed in the future.
{(devel)  unknown   }
```

This causes `$kustomize_load_restrictor_arg` to be set to:
```
--load-restrictor LoadRestrictionsNone
```

I believe this is the correct behavior.

Reply:
My output is
```
Flag --short has been deprecated, and will be removed in the future.
{(devel)  2023-09-11T18:05:37Z   }
```
and results in `$kustomize_load_restrictor_arg` to be  `--load_restrictor none`.

I'm not sure if it's a variation in how string comparisons are handled across Bash versions, locale settings, or something else, but regardless it I think the logic could be hardened.